### PR TITLE
AC-19 added a example request to a station to query properties

### PR DESCRIPTION
**Pull request by JonasPfeifer05**

### CODE REVIEW

**Review by leToxic — APPROVED** [review]
lgtm

File: justfile [hunk 1]
```diff
@@ -23,4 +25,10 @@ build:
     colcon build
 
 run-agent:
-    ros2 run fetch_agent agent
\ No newline at end of file
+    ros2 run fetch_agent agent
+
+all:
```
Comment (ToBinio) [inline justfile:30]: dont do that. These are not things you always want to run

you can add a build & run if you want but you dont want to isntall deps everytime and fix lints...

**Review by ToBinio — CHANGES_REQUESTED** [review]

File: justfile [hunk 2]
```diff
@@ -7,8 +7,10 @@ init:
     echo "source ./install/local_setup.bash" >> ~/.bashrc
 
 install:
-    python3 -m pip install uagents==0.17.1
-    python3 -m pip install git+https://github.com/Diplomarbeit-PGHFP-2024-2025/aca-protocols.git@ffbf199cde07a21d41ff08e14450fae9594082cd
+    rm -rd venv/lib/python3.12/site-packages/aca_protocols
+    rm -rd venv/lib/python3.12/site-packages/aca_protocols-0.1.0.dist-info
+    python3 -m pip install uagents==0.17.0
```
Comment (xFizzar) [inline justfile:12]: why not 0.17.1

**Review by xFizzar — CHANGES_REQUESTED** [review]

**Review by xFizzar — APPROVED** [review]
lgtm

**Review by ToBinio — APPROVED** [review]

**Pull request merged**